demo/demo.py

```diff
@@ -70,6 +72,24 @@ def get_parser():
     return parser
 
 
+def test_opencv_video_format(codec, file_ext):
+    filename = "test_file" + file_ext
+    writer = cv2.VideoWriter(
+        filename=filename,
+        fourcc=cv2.VideoWriter_fourcc(*codec),
+        fps=float(30),
+        frameSize=(10, 10),
+        isColor=True,
+    )
+    [writer.write(np.zeros((10, 10, 3), np.uint8)) for _ in range(30)]
+    writer.release()
+    if os.path.isfile(filename):
+        os.remove(filename)
+        return True
+    warnings.warn("x264 codec not available switching to mp4v")
```

Comment:
this warning shouldn't be here because it depends on what arguments this function is called with. either remove it or move it inside `__main__`

Reply:
Yeah, I thought of the same but thinking about the scope of this function I put it there but will change it as it may be confusing in the future.